.shiprc

```diff
@@ -1,6 +1,6 @@
 {
   "files": {
-    "auth0/__init__.py": []
```

Comment:
Removed this file as it states that it is now done on publish and not on creation of a ship PR